### ISSUE TITLE
ReferenceError: CKEDITOR is not defined 	  if (angular.isUndefined(CKEDITOR)) {

### ISSUE THREAD

**Issue by ipsitam** [esvit/ng-ckeditor#50]
I am getting  the above error when using it.

**Comment by MuadDib** [comment]
Same here. Trying to use it in VS2013 project. Manually added ng-ckeditor.js and ng-ckeditor.css to project.

**Comment by ipsitam** [comment]
do this before ur js gets loaded.

<script type="text/javascript">
	CKEDITOR.timestamp=null
	;
</script>

It worked for me.

**Comment by MuadDib** [comment]
Hmm, I appreciate the help but can you explain a bit more how and where you added it? 
CKEDITOR is not defined anywhere so I cannot just say CKEDITOR.timestamp as it's not defined.

**Comment by MuadDib** [comment]
Thank You, I figured it out... 
Didn't expect it to need cdEditor itself, but after including it it loads no problem. 

Too used to small directives you just add without dependency. 
Although it would've been nice it was mentioned Readme.md

**Comment by SayusiAndo** [comment]
I have ckEditor in my app, but I don't get this message. However, when I want to test my app with jasmin I get it. It makes fail my tests. What can I do with it?

**Comment by pshahidi** [comment]
Have you added them   to karma.conf.js (adjust path for you project struct.)
'lib/frontend/components/ng-ckeditor/libs/ckeditor/ckeditor.js',
'lib/frontend/components/ng-ckeditor/ng-ckeditor.min.js',

**Comment by Agent5** [comment]
@MuadDib: where did you add this? I can't figure it out.

**Comment by ChalangRasul** [comment]
I am also having this issue. I have it pointing to the ckeditor.js file in my bower_components directory and I made sure to add it below the angular declaration to make sure it has angular when running. I added ngCkeditor to my app.js file. This is the same process for all dependencies have added to my app. Except, this one causes an error whenever I try to load any page, and just gives me a blank screen. If I comment out the dependency in the app.js file, everything is fine, but re-adding it causes the error and blank screen.

**Comment by beazergood** [comment]
@chalangr12 a bit late to the party here but I don't suppose you figured out the issue in the end? I am loading ckeditor from the cdn (which works) but when adding to my karma config file it's complaining that CKEDITOR is not defined. I have included it as the first script that my tests load.

It loads fine for the application, it's just the tests that are failing.  Interestingly the test pass if I load it via bower but I can't get it to work in my production state when all my files get bundled.  Have tried setting the ckeditor base path to no avail.

So, i'm pretty much screwed one way or the other at the moment and found this thread on my googlings...

**Comment by ChalangRasul** [comment]
@beazergood Nope, I went with TinyMCE unfortunately, but I will have to figure it out probably.
